pkg/util/exec/error.go

```diff
@@ -41,6 +42,8 @@ func ExitStatus(err error) (int, bool) {
 		if ok {
 			return waitStatus.ExitStatus(), true
 		}
+	case kexec.CodeExitError:
+		return int(e.ExitStatus()), true
 
```

Comment:
Please remove this empty line to keep code consistency with existing `case`s.

Reply:
Removed.